server/apps/packages/package_service.py

```diff
@@ -82,11 +82,11 @@ def on_update(self, updates, original):
             self.extract_default_association_data(original, assoc)
 
     def on_updated(self, updates, original):
-        toAdd = {assoc.get(ITEM_REF): assoc for assoc in self._get_associations(updates)}
-        toRemove = [assoc for assoc in self._get_associations(original) if assoc.get(ITEM_REF) not in toAdd]
-        for assoc in toRemove:
+        to_add = {assoc.get(ITEM_REF): assoc for assoc in self._get_associations(updates)}
+        to_remove = [assoc for assoc in self._get_associations(original) if assoc.get(ITEM_REF) not in to_add]
```

Comment:
Since we are changing this, I would convert this `list` to a `set()` (faster checks for element presence).